### ISSUE TITLE
BodyReader shouldn't be a Singleton

### ISSUE THREAD

**Issue by BuildTestRich** [matthiasguentert/azure-appinsights-logger#16]
Hi, we've just started using the package but have found some problems with the latest version (v2.1.0).

We only get the original response body returned to the user on the very first request (after app restart).
All subsequent requests get the correct status code returned but a blank response body.

I believe it is because BodyReader has been changed to be a Singleton:

```
private static void AddBodyLogger(IServiceCollection services)
{
    services.AddSingleton<BodyLoggerMiddleware>();
    services.AddSingleton<IBodyReader, BodyReader>();
    services.AddSingleton<ITelemetryWriter, TelemetryWriter>();
    services.AddSingleton<ITelemetryInitializer, ClientIpInitializer>();
}
```

The `_originalResponseStreamReturned` flag will always be "true" after the first request.
Downgrading to v2.0.2 fixes the problem.

Also, separately, the max bytes field defaults to 80,000 bytes. This should probably be 8000 given that AppInsights custom properties only support 8,192 bytes.

`public int MaxBytes { get; set; } = 80000;`

**Comment by matthiasguentert** [comment]
Thanks for your bug report. Will investigate and provide a fix

**Comment by matthiasguentert** [comment]
Released 2.1.1, please let me know if things are working again.

**Comment by BuildTestRich** [comment]
Yep, that fixes it. Thanks for your help
